### PR TITLE
feat: Add docs to transfer a project / Remove an org

### PR DESCRIPTION
**Pull request by EvanPurkhiser**
Also renames the membrship page to 'Organization Management' to cover
more topics than just membership.

### CODE REVIEW

File: src/collections/_documentation/accounts/membership.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-title: Membership
+title: Organization Management
 sidebar_order: 1
 ---
 
-Membership in Sentry is handled at the organizational level. The system is designed so each user has a singular account which can be reused across multiple organizations (even those using SSO). Each user of Sentry should have their own account, and will then be able to set their own personal preferences in addition to receiving notifications for events.
+## Membership
 
-## Roles
+User membership in Sentry is handled at the organizational level. The system is
```
[the remark this answers is not in the document]
Comment (EvanPurkhiser) [reply src/collections/_documentation/accounts/membership.md]: This feels a little weird to use sentry as a company not a product in this sentence. I feel like I prefer "The system is designed so".

[the remark this answers is not in the document]
Comment (MimiDumpling) [reply src/collections/_documentation/accounts/membership.md]: It's in passive voice. The sentence needs a subject performing the action. But, if there's no way to make it active voice, that's okay. :)

[the remark this answers is not in the document]
Comment (MimiDumpling) [reply src/collections/_documentation/accounts/membership.md]: So, the question is, who designed the system?

[the remark this answers is not in the document]
Comment (EvanPurkhiser) [reply src/collections/_documentation/accounts/membership.md]: > So, the question is, who designed the system?

Sentry's Engineering team, but I think that's kind of implied

[the remark this answers is not in the document]
Comment (MimiDumpling) [reply src/collections/_documentation/accounts/membership.md]: Okie. That's a good enough reason to keep it in passive voice. ☺️

**Pull request merged**